app/controllers/resturants_controller.rb

```diff
@@ -14,6 +27,7 @@ def show
 
   # GET /resturants/new
   def new
+    @cuisines = Cuisine.all
```

Comment:
I needed it to render the add restaurant form. there's a dropdown selection of cuisines, and to generate it in react I needed the data from the model itself. Rails knows how to to it using the belongs_to connection with restaurants but I couldn't do it in react. was there another way to do that?